test/utils.py

```diff
@@ -50,12 +50,12 @@ def download_file(url, target):
     download.close()
 
 
-def LONG_TESTS():
-    LONG_TESTS = 0
+def LONG_TESTS() -> bool:
     # override LONG_TESTS with environment variable if available
-    if os.getenv("LONG_TESTS"):
-        LONG_TESTS = int(os.getenv("LONG_TESTS"))
-    return LONG_TESTS
+    env_var = os.getenv("LONG_TESTS")
+    if int(env_var) > 0:
```

Comment:
This is going to fail at runtime if environment variable is not set because in this case `getenv` returns None and it can't be parsed into `int`.
```suggestion
    if env_var and int(env_var) > 0:
```

You mentioned previously that CI errors out unless we do the `>` check. Can you point at such a run log? I only see this environment variable set to 0/1 in workflow files.

Reply:
Check #2363

Reply:
> Check #2363

@metabiswadeep, everything worked as expected there, the test wasn't skipped because `LONG_TESTS` environment variable was [set to '0'](https://github.com/intel/cve-bin-tool/actions/runs/3529534356/jobs/5920634217#step:14:6).

```
Run pytest -v --cov --cov-append --cov-report=xml test/test_cli.py test/test_cvedb.py
  pytest -v --cov --cov-append --cov-report=xml test/test_cli.py test/test_cvedb.py
  shell: /usr/bin/bash -e {0}
  env:
    ACTIONS: 1
    LONG_TESTS: 0
```

long_tests workflow doesn't **always** run long tests, only in some conditions (when env var is explicitly set for the run or when the haya14busa/action-cond GitHub Action sets up the `env.GIT_DIFF` variable. Notice how for the next commit `LONG_TESTS` was [set to '1'](https://github.com/intel/cve-bin-tool/actions/runs/3529569654/jobs/5931485411#step:14:6) by the Action in one of the steps above. It has nothing to do with the change to `> 0`.

So now with that mystery cleared, we can simplify this function even more! 😄
```py
def LONG_TESTS() -> bool:
    # override LONG_TESTS with environment variable if available
    env_var = os.getenv("LONG_TESTS")
    if env_var:
        return bool(int(env_var))
    return False
```